packages/esm-morgue-app/src/index.ts

```diff
@@ -34,3 +34,4 @@ export const morgueDashboardLink = getSyncLifecycle(
 );
 
 export const morgueEnrollWorkspace = getAsyncLifecycle(() => import('./workspaces/enrollBody.workspace'), options);
+export const autopsyForm = getAsyncLifecycle(() => import('./workspaces/autopsy.workspace'), options);
```

Comment:
```suggestion
import AutopysWorkspace from './workspaces/autopsy.workspace';
export const autopsyForm = getSyncLifecycle(AutopysWorkspace, options);
```
prefer using this for lazy loading of the modules